sources/src/main/java/com/google/solutions/jitaccess/web/ApiResource.java

```diff
@@ -312,6 +316,14 @@ public ActivationStatusResponse selfApproveActivation(
           .addLabels(le -> addLabels(le, activation))
           .addLabel("justification", request.justification)
           .write();
+
+        this.slackNotificationService.sendNotification(String.format(
```

Comment:
We don't send an emal in this case, but I think we could still use the same pattern:
1. Create a new class derived from Notification
2. Pass an instance to the NotificationService instance
3. Let the MailNotificationService ignore the call (it should automatically as it won't find an appropriate template)
4. Let the SlackNotificationService handle the call